core/src/main/java/hudson/model/UpdateSite.java

```diff
@@ -209,7 +209,7 @@ private FormValidation updateData(String json, boolean signatureCheck)
             }
         }
 
-        LOGGER.info("Obtained the latest update center data file for UpdateSource " + id);
+        LOGGER.finest("Obtained the latest update center data file for UpdateSource " + id);
```

Comment:
Maybe? I've grown so used to these messages over the years…

Since we would print errors when it doesn't work, this is probably fine.

Reply:
+1 for the change